changes-set.txt

```diff
@@ -43,13 +43,13 @@ proposed  syl5ibcom imbitridcom
 proposed  syl5ibr   imbitrrid
 proposed  syl5ibrcom imbitrridcom
 proposed  syl5bb    bitrid      compare to bitri or bitrd
-proposed  syl5rbb   bitridcom
+proposed  syl5rbb   bitr2id
```

Comment:
Is it OK to add `compare to bitr2i or bitr2d` to the notes column? I find those notes helpful when I'm preparing pull requests to actually rename some of these.